pkg/project/registry/project/proxy/proxy.go

```diff
@@ -111,8 +111,12 @@ func (s *REST) Watch(ctx apirequest.Context, options *metainternal.ListOptions)
 var _ = rest.Getter(&REST{})
 
 // Get retrieves a Project by name
-func (s *REST) Get(ctx apirequest.Context, name string) (runtime.Object, error) {
-	namespace, err := s.client.Get(name, metav1.GetOptions{})
+func (s *REST) Get(ctx apirequest.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
+	opts := metav1.GetOptions{}
+	if options != nil {
```

Comment:
Good catch. @ncdc I remember I saw some unguarded dereferencing of options in some of your PRs, maybe also in mine.